### PR TITLE
Fix: Escape glyph hex in xaml

### PR DESCRIPTION
**Pull request by HO-COOH**
## Description
Escape glyph hex in xaml sample

## Screenshots (if appropriate):
image
->
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by niels9001 — APPROVED** [review]
Tested, works well!

**Comment by karkarl** [comment]
I reconfigured the CI pipeline from Azure side. I'm unable to trigger a manual build as it's a forked branch, do you mind pushing a "whitespace" commit to see if it triggers a build?

**Comment by HO-COOH** [comment]
@karkarl Done

File: WinUIGallery/ControlPages/InfoBadgePage.xaml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     mc:Ignorable="d">
 
     <StackPanel>
-        <local:ControlExample x:Name="Example1" HeaderText="InfoBadge embedded in NavigationView" HorizontalContentAlignment="Stretch" >
```
Comment (chingucoding) [inline WinUIGallery/ControlPages/InfoBadgePage.xaml:23]: nit: Unrelated to PR

**Review by chingucoding — APPROVED** [review]

**Pull request merged**